lib/note-content-editor.tsx

```diff
@@ -132,6 +133,11 @@ class NoteContentEditor extends Component<Props> {
         }
       }, SPEED_DELAY);
     }
+    if (prevProps.lineLength !== this.props.lineLength) {
+      setTimeout(() => {
+        window?.editor.layout();
```

Comment:
should be `window.editor?.layout()`

Reply:
If the editor exists on window layout will exist. This checks to make sure editor is there.

Reply:
Updated to: window?.editor?.layout();

Reply:
you don't need `window?.` because `window` will always exist, hence `window.editor?.` not `window?.editor`

`window?.editor.layout()` says "if indeed `window` exists then I assert that `editor` will always exist on it and `layout()` will be a method on `editor`"

`window.editor?.layout()` says "if indeed `editor` exists on `window` then I assert that `layout()` will be a method on it"

Reply:
🤦 yeah...